### PR TITLE
implement NPL selection, simplify selection

### PR DESCRIPTION
**Pull request by jhgoh**
Implement NPL selection
Simplify routines, event selection logic

### CODE REVIEW

**Comment by jhgoh** [comment]
@jeongsumin @yhlee52 plz check.

File: TopAnalysis/src/FCNCTriLeptonCppWorker.cc [hunk 1]
```diff
@@ -151,10 +148,15 @@ bool FCNCTriLeptonCppWorker::isNPElectron(const unsigned i) const {
...
   if ( std::abs(scEta) > 1.4442 and std::abs(scEta) < 1.566 ) return false;
   //nanoAOD object -> Electron_vidNestedWPBitmapSum16 (total 10 cuts in here by bitmap, each cut has 3 bit)
-  //int vidNestedWPBitmapSum16 = in_Electrons_vidNestedWPBitmapSum16->At(i);
-  //if ( ! ( vidNestedWPBitmapSum16 > ... ) ) return false;
+  const unsigned vidBitmap = in_Electrons_vidBitmap->At(i);
+  for ( unsigned i=0; i<10; ++i ) {
+    const unsigned vidBit = (vidBitmap & (0x7<<(3*i))); // 0x7 = 111 in the binary representation
+    const bool isLooseOnThisBit = vidBit >= 2; // 3 bits to store cuts. 0:fail, 1:veto, 2:loose, 3:medium, 4:tight
+    //if ( i != 5 and !isLooseOnThisBit or i == 5 and  isLooseOnThisBit ) return false;
+    if ( (i == 5) == (isLooseOnThisBit) ) return false;
```
Comment (yhlee52) [inline TopAnalysis/src/FCNCTriLeptonCppWorker.cc]: The condition will be (i == 2).
isocut is 3rd part from lower bits.

Comment (yhlee52) [reply TopAnalysis/src/FCNCTriLeptonCppWorker.cc]: Also, the isolation criteria is not inverted loose but "inverted tight" criteria. It means, isolation criteria should be fail, veto, loose and medium.

Comment (jhgoh) [reply TopAnalysis/src/FCNCTriLeptonCppWorker.cc]: fixed.

**Pull request merged**